### DISCUSSION TITLE
Use Mockoon to mock create and update services

### DISCUSSION THREAD

**Discussion by parulbhagat-soprabanking** [mockoon/mockoon#1091] [Help]
Hi ,

Can mockoon be used to mock create and update services? Our requirement:

1) Mock a create service that creates data in the database mentioned.
2) Mock an update service that updates data in the database mentioned.
3) If not mockoon, then is there any other toll that can help us for this requirement ?

Thanks !

**Comment by 255kb** [comment]
Hi, 

The easiest is to use a [CRUD enpoint](https://mockoon.com/docs/latest/api-endpoints/crud-routes/). It's basically a replacement for json-server. It lets you call the usual REST endpoints like GET /resources/:id, PUT /resources/:id, etc. It manipulates the data bucket of your choice. 
It will be the same mock API though. Data buckets are not shared between environments. 
To solve this, you could create two CRUD endpoints with different paths: 
`/service1/resource` and `/service2/resource`. You set them to both manipulate the same data bucket and it should work.

**Reply by parulbhagat-soprabanking** [reply]
Thanks for the reply, I will try this and get back here if I face any issues.
